web/src/lib/utils/cookies.ts

```diff
@@ -0,0 +1,30 @@
+import { browser } from '$app/environment';
+
+const IsAuthenticatedCookieName = 'immich_is_authenticated';
+
+export const setAuthCookie = (): void => {
```

Comment:
set and remove should just be deleted as this should be implemented server-side.